todo_app/viewmodel.py

```diff
@@ -22,11 +22,11 @@ def show_all_done_items(self):
     def recent_done_items(self):
         all_done_items = [item for item in self._items if item.status == 'Done']
         today = datetime.date.today()
-        recent_done_items = [item for item in all_done_items if item.lastmodifieddate >= today]    
+        recent_done_items = [item for item in all_done_items if item.lastmodifieddate.day >= today.day]    
         return recent_done_items
     @property
     def older_done_items(self):
         all_done_items = [item for item in self._items if item.status == 'Done']
         today = datetime.date.today()
-        older_done_items = [item for item in all_done_items if item.lastmodifieddate < today]    
+        older_done_items = [item for item in all_done_items if item.lastmodifieddate.day < today.day]    
```

Comment:
This change breaks your app - if an item was modified on 25th May, and we check on 2nd June, then since 25 > 2, we consider the item to be done recently (today!)
Reverting to just compare the dates directly is a better solution